lopper/lops/lop-domain-linux-a53-prune.dts

```diff
@@ -56,6 +56,7 @@
                       code = "
                           # Check for domain node
                           from domain_access import update_mem_node
+                          from baremetalconfig_xlnx import get_label
```

Comment:
We now have get_label() in two assists.  We can look at moving it into a lopper library to avoid the duplication.

Let me ask a question though (not related to this review, but for my information in consolidating the routine).

What is get_label() actually doing ? It is checking the node __props__ dictionary, which is an ordered dict of property names to LopperProp objects.

But yet the list comprehension / evaluation variables are 'label' and 'node_abs'. The node_abs[0], will get the property as a string (in the first chunk), so that would work.

I assume that the symbol node has properties that are name -> node path ? And that this is looking for a LopperProperty that points to the node being compared to.

The variables are a bit oddly named, but I hope I've got the intent correct.

Reply:
Yes @zeddii  your understanding is correct, please let me know once the changes are available in the lopper library  will remove the get_label API and will update source code according to new implementation